lib/Wikia/src/Service/User/Preferences/PreferenceModule.php

```diff
@@ -14,12 +15,17 @@
 
 class PreferenceModule implements Module {
 	const PREFERENCE_CACHE_VERSION = 3;
+	const PREFERENCE_CACHE = false;
 
 	public function configure( InjectorBuilder $builder ) {
 		$builder
 			->bind( PreferenceService::class )->toClass( PreferenceServiceImpl::class )
 			->bind( PreferencePersistence::class )->toClass( PreferencePersistenceSwaggerService::class )
 			->bind( PreferenceServiceImpl::CACHE_PROVIDER )->to( function() {
+				if ( !self::PREFERENCE_CACHE ) {
```

Comment:
Hm sorry I didn't see this comment until now.

Yah I agree that this probably isn't the best way to do this, but this isn't intended to live in the codebase for long. We will keep this until we're satisfied that the preference service can handle the traffic load, and then remove the `PREFERENCE_CACHE` constant as well as this conditional. Does that seem ok?